shlr/rizin-shell-parser/corpus/simple_commands.txt

```diff
@@ -161,7 +161,7 @@ pf???
   (help_command (cmd_identifier))
   (help_command (cmd_identifier))
   (help_command (cmd_identifier))
-  (help_command (cmd_identifier))
+  (arged_command (cmd_identifier))
```

Comment:
Is this intended change?

Reply:
Yes. It is due to pr done by @wargio, which makes ?* Behave like a arged_command, right @wargio?

Reply:
yes